bufferserver/src/main/java/com/datatorrent/bufferserver/server/Server.java

```diff
@@ -73,9 +73,12 @@
   private InetSocketAddress address;
   private final ExecutorService serverHelperExecutor;
   private final ExecutorService storageHelperExecutor;
+  private boolean backPressureEnabled;
 
   private byte[] authToken;
 
+  private static final boolean BACK_PRESSURE_ENABLED = !Boolean.getBoolean("BUFFERSERVER.BACK_PRESSURE_DISABLE");
```

Comment:
Can it be final static boolean of the DataList class itself to avoid passing the parameter over few classes that is static in nature? For the property name, I'd suggest to use "com.datatorrent.bufferserver.backpressure.disable" or "com.datatorrent.bufferserver.internal.DataList.backpressure.disable".